requirements.txt

```diff
@@ -1,21 +1,51 @@
+appnope==0.1.0
+astroid==2.0.4
 atomicwrites==1.2.1
 attrs==18.2.0
+backcall==0.1.0
+certifi==2018.10.15
+chardet==3.0.4
 coverage==4.5.1
+coveralls==1.5.1
+decorator==4.3.0
 Django==2.1.2
 django-cors-middleware==1.3.1
 django-extensions==2.1.3
-gunicorn==19.9.0
-whitenoise==4.1
+django-nose==1.4.6
+django-taggit==0.23.0
+django-taggit-serializer==0.1.7
 djangorestframework==3.9.0
+docopt==0.6.2
+flake8==3.6.0
```

Comment:
The number of packages in the `requirements.txt` has massively grown. Could they be dependencies of some package(s) you installed ? or is there a possibility that you forgot and did a `pip freeze` in a global environment?

Reply:
I installed them, I did not pip freeze in a global environment, they are dependencies